src/olympia/hero/models.py

```diff
@@ -0,0 +1,16 @@
+from django.conf import settings
+from django.db import models
+
+from olympia.amo.models import ModelBase
+from olympia.discovery.models import DiscoveryItem
+
+
+class PrimaryHero(ModelBase):
+    image = models.CharField(max_length=255)
+    background_color = models.CharField(max_length=7)
```

Comment:
There was some discussion today about how we should record this color choice. It seemed like using css color variables, which will be defined in an .scss file, would be a good idea, but no decision was made. I take it that this format assumes a hex color? I see no reason why that wouldn't work for the front end to consume but before we commit to a restrictive format we should perhaps decide what's going to work best.

Reply:
It could change depending on the answer to the question in the doc, but I was assuming it would be a hex color value, yeah.  Return color names/id (which would have to be kept in sync) seems more complicated than showing swatches in the admin and returning the hex color.

Reply:
I don't think we can merge this until https://github.com/mozilla/addons-server/issues/11830#issuecomment-513230982 is answered because it'll be a pain to change how colours are represented.

Reply:
added https://github.com/mozilla/addons-server/pull/11858/commits/acaeb79aa6532653fdefda2c308bec7062c07268 for the gradient pair
(+ https://github.com/mozilla/addons-server/pull/11858/commits/53f56d7813cb0b4868c1ef43be7184c4b5584028 for the docs)